packages/core/content-manager/server/tests/content-manager/content-types.test.e2e.js

```diff
@@ -16,6 +19,14 @@ const restart = async () => {
   rq = await createAuthRequest({ strapi });
 };
 
+const transformLayout = (layout, partialNewLayout) => {
+  return partialNewLayout.reduce((acc, row, rowIndex) => {
+    acc[rowIndex] = row.map((column, columnIndex) => merge(acc[rowIndex][columnIndex], column));
+
+    return acc;
+  }, layout);
+};
+
```

Comment:
is `transformLayout` equivalent to lodash merge function?

![Capture d’écran 2022-02-23 à 12 02 15](https://user-images.githubusercontent.com/11708220/155307009-645895cd-8f41-4632-adbd-5b66dfff8b68.png)

(I should have deep clone in the second call but it shouldn't change the result)

Reply:
Maybe better than `cloneDeep` :
```js
_.merge([], FIXTURE_DEFAULT_LAYOUT, transformation)
```

Reply:
That's a very good suggestion. I didn't know `_.merge` can operate on arrays. Very nice. Thanks!